# Table of Content

- [Table of Content](#table-of-content)
- [0-General](#0-general)
  - [Introduction](#introduction)
  - [Study Objectives](#study-objectives)
  - [Study Significance](#study-significance)
  - [Methods - Study Design](#methods---study-design)
  - [Methods - Outcome Measures](#methods---outcome-measures)
  - [Sample Size Calculation Rationale](#sample-size-calculation-rationale)
  - [Simulation-Based Sample Size Calculation for ANCOVA](#simulation-based-sample-size-calculation-for-ancova)
- [1-Sample Size Calculation](#1--sample-size-calculation)
- [2-Conclusion](#2-conclusion)

# 0-General
[Back to Table of Content](#table-of-content)
# Sample Size Calculation for the Nature-Boost Randomized Controlled Trial

## Introduction
[Back to Table of Content](#table-of-content)

The Nature-Boost project aims to address the rising challenges in long-term care facilities by enhancing the health and relationship between caregivers and residents through innovative, nature-based interventions. The project recognizes the dual burdens on both residents and staff due to increased demand for long-term care services and chronic workforce shortages in caregiving roles. In particular, residents often experience reduced mobility, social isolation, and sensory deprivation, which can impact sleep quality, mood, and overall well-being. Caregivers also face high levels of stress and burnout, contributing to a challenging work environment.

To counter these challenges, the Nature-Boost project proposes immersive, multi-sensory interventions replicating nature experiences. These include:
1. **Olfactory stimuli**: Essential oil blends simulating the scent of the Bavarian forest.
2. **Visual stimuli**: VR simulations showing forest scenes.
3. **Auditory stimuli**: Natural sounds, such as forest ambiance.


## Study Objectives
[Back to Table of Content](#table-of-content)

The Nature-Boost study aims to test the health-promoting effects of virtual and sensory nature experiences for both residents and caregiving staff in long-term care settings. Specifically, it examines:
1. **Primary Objective**: Assess whether immersive nature-based experiences improve residents' sleep quality.
2. **Secondary Objectives**: Evaluate improvements in residents' overall well-being, mood, loneliness, cognitive function, and emotional exhaustion. Additionally, assess changes in caregivers' perceived stress and burnout.


## Study Significance
[Back to Table of Content](#table-of-content)

The anticipated outcomes of Nature-Boost align with an urgent need for effective mental and physical health interventions in long-term care. By exploring whether digitally simulated nature experiences can improve sleep and reduce stress for those unable to access outdoor environments directly, the study could pave the way for scalable, accessible solutions for both residents and caregivers.


## Methods - Study Design
[Back to Table of Content](#table-of-content)

This study will use a randomized controlled trial (RCT) design with four intervention groups:
- **Experimental Group 1**: Participants receive an immersive sensory experience before sleep, combining forest-scented essential oils, VR forest images, and natural forest sounds.
- **Experimental Group 2**: Participants experience VR images and natural sounds of the forest, without scent stimuli.
- **Experimental Group 3**: Participants are exposed to forest-scented essential oils alone.
- **Control Group**: Participants receive a neutral intervention, such as a non-nature-related video, without accompanying scents or sounds.

## Methods - Outcome Measures
[Back to Table of Content](#table-of-content)

- **Primary Outcome**: Sleep quality, assessed using the Pittsburgh Sleep Quality Index (PSQI).
- **Secondary Outcomes**:
  - Residents' well-being (WHO-5 Well-Being Index), loneliness (UCLA Loneliness Scale), mood (Hospital Anxiety and Depression Scale), and cognitive function.
  - Caregivers' stress (Perceived-Stress-Scale) and burnout risk (Burnout Assessment Tool).


## Sample Size Calculation Rationale
[Back to Table of Content](#table-of-content)

For this randomized controlled trial (RCT), we aim to detect differences in PSQI scores across four intervention groups. Due to the inclusion of baseline covariates (e.g., baseline PSQI scores, age, sex, medication use), an ANCOVA model is more suitable than ANOVA, as it adjusts for these covariates and provides increased precision.

To determine an appropriate sample size for detecting differences in PSQI scores across the four groups, we consider the study by Chang, Wang, and Yu (2014, PMID: 25394270), which investigated the effects of VR interventions on sleep quality in elderly participants using the PSQI.

Reference Study Data (Chang, Wang, and Yu 2014)

The study by Chang, Wang, and Yu (2014, PMID: 25394270) provides benchmark values for PSQI changes in a control and an experimental group, as shown below:

| Group          | Baseline PSQI Mean | Post-Treatment PSQI Mean | Standard Deviation (SD) |
|----------------|--------------------|--------------------------|--------------------------|
| Control        | 5.60              | 6.20                    | 1.93                    |
| Experimental   | 5.44              | 4.56                    | 1.67                    |

Chang et al.’s findings indicate a significant improvement in PSQI for the experimental group (mean reduction of ~0.88 points) compared to the control group (mean increase of ~0.60 points). We use these values as assumptions for the expected impact of Nature-Boost's VR interventions on PSQI.


## Simulation-Based Sample Size Calculation for ANCOVA
[Back to Table of Content](#table-of-content)

To accommodate the complexity of ANCOVA and covariate adjustments, we calculate the effect size step by step and conduct a power analysis using these parameters.

# 1- Sample Size Calculation
[Back to Table of Content](#table-of-content)

In [54]:
# Load necessary package
if (!requireNamespace("Superpower", quietly = TRUE)) install.packages("Superpower")
library(Superpower)

In [57]:
# Calculate the effect size (f^2) step by step
# Using the ANCOVA F-statistic (F = 4.74, df_between = 1, df_error = 17)
F_stat <- 4.74
df_between <- 1
df_error <- 17
eta_squared <- (F_stat * df_between) / (F_stat * df_between + df_error)
cat("Eta squared:", eta_squared, "\n")

Eta squared: 0.2180313 


In [58]:
# Convert eta^2 to f^2
# Formula: f^2 = eta^2 / (1 - eta^2)
f2 <- eta_squared / (1 - eta_squared)
cat("Effect size (f^2):", f2, "\n")

Effect size (f^2): 0.2788235 


In [59]:
# Representative SD for PSQI scores based on experimental group posttest
sd <- 1.67

In [60]:
# Number of covariates (baseline PSQI, age, medication use)
n_cov <- 3

In [61]:
# Proportion of variance explained by covariates
r2 <- 0.25

In [62]:
# Significance level and power
alpha_level <- 0.05  # Standard significance threshold
beta_level <- 0.2    # 1 - desired power (80%)

In [63]:
# Expected attrition rate
attrition_rate <- 0.30

In [64]:
# Perform power calculation using the Superpower package
# Shieh's exact method for ANCOVA
results <- Superpower::power_oneway_ancova(
  mu = mu,                      # Group means
  n_cov = n_cov,                # Number of covariates
  sd = sd,                      # Representative SD
  r2 = r2,                      # Proportion of variance explained by covariates
  alpha_level = alpha_level,    # Significance level
  beta_level = beta_level,      # Desired beta level
  round_up = TRUE,              # Round up to nearest whole participant
  type = "exact"                # Use Shieh's exact method
)

In [65]:
# Display the results
cat("\nPower Calculation Results:\n")
print(results)


Power Calculation Results:

     Power Calculation for 1-way ANCOVA 

            dfs = 3, 165
              N = 172
              n = 43, 43, 43, 43
          n_cov = 3
             mu = 5.60, 4.56, 5.00, 5.20
             sd = 1.67
             r2 = 0.25
    alpha_level = 0.05
     beta_level = 0.1939559
          power = 80.60441
           type = exact



In [66]:
# Adjust sample size for attrition
# Formula: Adjusted N = Total N / (1 - attrition rate)
adjusted_sample_size <- ceiling(results$N / (1 - attrition_rate))
cat("\nAdjusted Total Sample Size (accounting for 30% attrition):", adjusted_sample_size, "\n")


Adjusted Total Sample Size (accounting for 30% attrition): 246 


# 2-Conclusion
[Back to Table of Content](#table-of-content)

The objective of this analysis was to estimate the minimum sample size required per group to achieve adequate statistical power for detecting differences in PSQI (Pittsburgh Sleep Quality Index) scores across four intervention groups in the Nature-Boost randomized controlled trial. Given the study’s design and the inclusion of baseline PSQI scores as a covariate, an ANCOVA model was chosen as the primary method for analysis.

We employed an analytical power analysis using the effect size calculated from Chang et al. (2014) and parameters specific to the Nature-Boost study. A target power of 80% was set, which is typically sufficient for clinical studies. Due to the elderly participant population, we accounted for a 30% dropout rate to ensure adequate power despite potential participant loss.

The results from the power analysis indicated that a sample size of 43 participants per group would be required to achieve the target power of 80% without attrition. However, with a 30% attrition adjustment, the final recommended sample size increased to 62 participants per group. For the four-group design, this yields a total sample size of 246 participants. This adjusted sample size is expected to provide sufficient power, accounting for the elevated dropout rate, which is anticipated due to participants' advanced age and the physical constraints some may face with VR and sensory technology.

In conclusion, recruiting 246 participants in total (62 per group) will allow the Nature-Boost study to detect meaningful differences between intervention groups in terms of sleep quality, even if a significant proportion of participants withdraw. This ensures that the study maintains robust statistical power and can draw reliable conclusions about the impact of nature-based interventions on sleep quality and well-being in long-term care residents.
